doc/js-api/remotestorage.rst

```diff
@@ -16,6 +16,43 @@ example::
      cordovaRedirectUri: 'https://app.wow-much-app.com' // defaults to undefined
    });
 
+Here is a complete list of the default options (set in ``src/config/js``), all
+of which can be overridden by calling the constructor with a configuration
+object like above::
+
+   {
+     // Enable remoteStorage logging
+     logging: false,
+     // Change Events that are enabled. See the BaseClient API for more information
+     changeEvents: {
+       local:    true,
+       window:   false,
+       remote:   true,
+       conflict: true
+     },
+     // enable caching
+     cache: true,
+     // disable specified modules
+     disableFeatures: [],
+     // timeout for the Webfinger lookup, discovering a connecting user's storage details
+     discoveryTimeout: 10000,
+     // sync interval when the application is in the foreground
+     syncInterval: 10000,
+     // sync interval when the application is in the background
+     backgroundSyncInterval: 60000,
+     // initial value for the internal state of the app being in the background
+     isBackground: false,
+     // set a redirect URI for Cordova apps
+     cordovaRedirectUri: undefined,
+     // timeout for network requests
+     requestTimeout: 30000,
+     // extra ES6 modules to load
+     modules: []
+   };
+
+For more information about the Change Events, see
+:doc:`the BaseClient API doc</js-api/base-client>`.
```

Comment:
This is documented elsewhere. No need to link to there from here, but important to link to here from there.